src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java

```diff
@@ -286,11 +286,11 @@ public final MethodResult getChemical(IArguments arguments) throws LuaException
     @LuaFunction(mainThread = true)
     public final MethodResult listItems(IArguments arguments) throws LuaException {
         if (!isAvailable())
-            return notConnected();
+            return notConnected(Collections.emptyList());
```

Comment:
for lists, since it is not possible to return a half completed list, I'd recommend use `null` instead of emptyList. So user can check error easier, as well as its not creating extra lua tables

Reply:
I agree. But we should do that also for the other errors in these functions like these
https://github.com/IntelligenceModding/AdvancedPeripherals/blob/0d93afa17b1f82f2ee36f32bddc541ddf80b52c7/src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java#L293

Reply:
yes